README.md

```diff
@@ -29,48 +29,12 @@ This SDK supports:
 * UIKit
 * SwiftUI
 
-## Access Token
+## ClientID
 
-The PayPal SDK uses access tokens for authentication.
+The PayPal SDK uses clientID for authentication.
 
 > The following example can be adapted to any server-side language/framework of your choice. We use command-line curl requests to demonstrate the overall composition of the Access Token HTTP request.
 
-To create an access token:
-
-1. Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to obtain a client ID and secret from the PayPal Developer site.
-1. Make an HTTP request with Basic Authentication using client ID and secret to fetch an access token:
-
-**Request**
-```bash
-# for LIVE environment
-curl -X POST https://api.paypal.com/v1/oauth2/token \
--u $CLIENT_ID:$CLIENT_SECRET \
--H 'Content-Type: application/x-www-form-urlencoded' \
--d 'grant_type=client_credentials&response_type=token&return_authn_schemes=true'
-
-# for SANDBOX environment
-curl -X POST https://api.sandbox.paypal.com/v1/oauth2/token \
--u $CLIENT_ID:$CLIENT_SECRET \
--H 'Content-Type: application/x-www-form-urlencoded' \
--d 'grant_type=client_credentials&response_type=token&return_authn_schemes=true'
-```
-
-:warning:&nbsp;Make sure the environment variables for `CLIENT_ID` and `CLIENT_SECRET` are set.
-
-**Response**
-
-```json
-{
-  "scope": "...",
-  "access_token": "<ACCESS_TOKEN>",
-  "token_type": "Bearer",
-  "app_id": "...",
-  "expires_in": 32400,
-  "nonce": "..."
-}
-```
-
-Use the value for `access_token` in the response to create an instance of `CoreConfig` to use with any of the SDK's feature clients.
 
```

Comment:
Do we still need this so that merchants can to make calls to create, authorize/capture orders on their servers?

Reply:
I don't think we need to include it in our docs. We can rely on this being documented in the PayPal docs for the server side authorize/capture call. It was only documented here since it was necessary to use the SDK previously.